docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc

```diff
@@ -93,8 +93,8 @@ server factories.
 the SASL server factory is an aggregation of other SASL server
 factories.
 
-|configurable-http-server-mechanism-factory |A SASL server factory
-definition where the SASL server factory is an aggregation of other SASL
+|configurable-http-server-mechanism-factory |An HTTP server factory
```

Comment:
Since this change is unrelated to this RFE and would be good to get included soon, what do you think about creating a good first issue for this update instead?